src/ark/phenotyping/pixie_preprocessing.py

```diff
@@ -240,7 +244,7 @@ def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
         os.mkdir(os.path.join(base_dir, subset_dir))
 
     # create variable for storing 99.9% values
-    quant_dat = pd.DataFrame()
+    quantile_path = os.path.join(base_dir, data_dir, "quantile_data.csv")
```

Comment:
Again pretty inconsequential, but can we change the filename to "channel_norm_post_rowsum_perfov.csv", just so the name is a little more descriptive (and matches the feather file name)?